pkg/monitor.go

```diff
@@ -58,35 +62,38 @@ func New(config *Config, zapLogger *zap.Logger) *Monitor {
 			continue
 		}
 
-		err = relay.GetStatus()
+		err = relay.Status(context.TODO())
```

Comment:
Note: we use contexts extensively and most exported funcs take in a context as a first arg. It would be ideal to revisit how we want to use contexts in this project and use idiomatic context cancelation for cleaning up routines